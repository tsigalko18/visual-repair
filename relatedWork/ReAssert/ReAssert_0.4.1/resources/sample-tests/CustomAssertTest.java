import static org.junit.Assert.assertTrue;
import static org.junit.Assert.assertFalse;

import org.junit.Test;

public class CustomAssertTest {
	
	@Test
	public void testWithCustomAssert() {
		myAssertTrue(false);
	}
	
	@Test
	public void testWithoutCustomAssert() {
		assertTrue(false);
	}
	
	public void myAssertTrue(boolean value) {
		assertTrue(value);
	}

	public void myAssertFalse(boolean value) {
		assertFalse(value);
	}
	
}
