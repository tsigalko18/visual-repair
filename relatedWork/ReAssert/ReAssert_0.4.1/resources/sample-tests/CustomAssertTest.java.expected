import static org.junit.Assert.assertTrue;
import static org.junit.Assert.assertFalse;

import org.junit.Test;

public class CustomAssertTest {
	
	@Test
	public void testWithCustomAssert() {
		myAssertFalse(false);
	}
	
	@Test
	public void testWithoutCustomAssert() {
		assertFalse(false);
	}
	
	public void myAssertTrue(boolean value) {
		assertTrue(value);
	}

	public void myAssertFalse(boolean value) {
		assertFalse(value);
	}
	
}
